storage/tests/unit/test_bucket.py

```diff
@@ -2146,7 +2178,7 @@ def test_set_iam_policy_w_user_project(self):
         USER_PROJECT = "user-project-123"
         PATH = "/b/%s" % (NAME,)
         ETAG = "DEADBEEF"
-        VERSION = 17
+        VERSION = 1
```

Comment:
Do we have tests that try with version == 3?

Reply:
That would be covered by the `api_core` unit tests, since handling incoming version is more of its domain.

We do have tests for `requested_policy_version` above:
https://github.com/googleapis/google-cloud-python/pull/9989/files#diff-057063ca79c402a49be445a6d453c448R2108